src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -531,6 +532,17 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 				};
 				const newPart = this.renderChatContentPart(data, templateData, context);
 				if (newPart) {
+
+					if (this.rendererOptions.renderDetectedCommandsWithRequest
+						&& isRequestVM(element) && element.agentOrSlashCommandDetected && element.slashCommand
+						&& data.kind === 'markdownContent' // TODO this is fishy but I didn't find a better way to render on the same inline as the MD request part
```

Comment:
I think you should push a new `IChatRendererContent` into `value`, then the rendering can work a bit more normally, does that make sense?

Reply:
Yeah, but I don't know how to make it so that it renders inline with the MD content, e.g the latter is a block-element (`div`) and this element wants to be inline with in